packages/ref-data-uploaders/txc-uploader/txc_processor.py

```diff
@@ -557,6 +621,14 @@ def write_to_database(
                 )
                 return False
 
+            if not file_has_vehicle_journeys:
+                db_connection.rollback()
```

Comment:
Will vehicle journey be available in all the xml files? If yes has this been confirmed?

Reply:
Steve's confirmed there should always be vehicle journeys yeah

Reply:
as discussed